application/server/notification/scan_notifier.go

```diff
@@ -158,7 +158,7 @@ func (n *scanNotifier) appendCodeIssues(scanIssues []lsp.ScanIssue, folderPath s
 		}
 
 		scanIssues = append(scanIssues, lsp.ScanIssue{
-			Id:       issue.ID,
+			Id:       additionalData.Key,
```

Comment:
Why are we changing code issues as well?

Reply:
See the PR description:
> We've assumed that it's not surfaced in UI and changed in [here](https://github.com/snyk/snyk-ls/pull/216/files#diff-c487190192b10556a66dd6213400d3084215c69eb2c4f547285fe7b4ab61c0f4), however these are reported in problem views as issue IDs (see screenshot attached).